java/change-notes/2021-03-05-commons-object-utils.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Add models for ObjectUtils functions in the Apache Commons-Lang library. This may lead to more results from any dataflow query where traversal of ObjectUtils functions means we can now complete a path from a source of tainted data to a corresponding sink.
```

Comment:
Really minor, but it looks like the project itself does not use a hyphen, see https://commons.apache.org/proper/commons-lang/
```suggestion
* Added models for `ObjectUtils` methods in the Apache Commons Lang library. This may lead to more results from any dataflow query where traversal of `ObjectUtils` methods means we can now complete a path from a source of tainted data to a corresponding sink.
```